coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentControllerBase.java

```diff
@@ -394,11 +394,13 @@ protected ModelAndView proposalDevelopmentPageSave(ProposalDevelopmentDocumentFo
     protected ModelAndView narrativePageSave(ProposalDevelopmentDocumentForm form, boolean canEdit) throws Exception {
         ProposalDevelopmentDocument document = (ProposalDevelopmentDocument) getDocumentService().getByDocumentHeaderId(form.getDocument().getDocumentNumber());
         if (canEdit) {
-            if (new ProposalDevelopmentDocumentRule().processAttachmentRules(form.getProposalDevelopmentDocument())) {
+            if ((new ProposalDevelopmentDocumentRule().processAttachmentRules(form.getProposalDevelopmentDocument()))
+                    && (new ProposalDevelopmentDocumentRule().processPersonnelAttachmentDuplicates(form.getProposalDevelopmentDocument()))) {
             	form.getProposalDevelopmentAttachmentHelper().handleNarrativeUpdates(form, document);
             	form.getProposalDevelopmentAttachmentHelper().handleInstAttachmentUpdates(form, document);
             	form.getProposalDevelopmentAttachmentHelper().handlePersonBioUpdates(form, document);
                 document.getDevelopmentProposal().setProposalAbstracts(form.getDevelopmentProposal().getProposalAbstracts());
+                document.setDocumentNextvalues(form.getProposalDevelopmentDocument().getDocumentNextvalues());
```

Comment:
So documentNextValues is used across prop dev. Person, person degrees, special review and attachments. By overwriting the value you are saying that the attachment section of the document owns documentNextvalues, when it doesn't. But the inverse isn't true as well, in that the other locking section doesn't own it either. So since both sections can modify this list, we need to do something smarter than just overwriting it. I think we need to merge the values basically.